src/carousel/carousel.scss

```diff
@@ -120,6 +124,8 @@
     &.iui-active::after {
       @include themed {
         background-color: t(iui-color-foreground-primary);
+        border-color: t(iui-color-foreground-primary);
+        box-shadow: inset 0 0 0 1px t(iui-color-background-1);
```

Comment:
box-shadow will not work with forced colors mode.

Reply:
I've added `@media (forced-colors: active) { border-width: $iui-icons-small * 0.5; }`.
![Screen Shot 2022-03-23 at 2 11 50 PM](https://user-images.githubusercontent.com/849817/159767589-8824a4e5-7f82-46c1-87dd-5c4bd3261c6e.png)